src/draw/lv_draw_buf.c

```
@@ -19,6 +19,8 @@
 #define font_draw_buf_handlers LV_GLOBAL_DEFAULT()->font_draw_buf_handlers
 #define image_cache_draw_buf_handlers LV_GLOBAL_DEFAULT()->image_cache_draw_buf_handlers
 
+#define draw_buf_get_handler(drawbuf) ((drawbuf)->handlers ? (drawbuf)->handlers : &default_handlers)
```

Seems like `->handlers` can never be `NULL`.

---

IMO if it's `NULL` it's rather an error that shouldn't be silently replaced by the defaults.

---

There's hacky code assembles `lv_draw_buf_t` manually and could have NULL handlers. I'll fix them and add assert directly.